Editor/Urls.cs

```diff
@@ -29,7 +29,7 @@ internal static class Urls
         public const string S3BootstrapHelp = "https://docs.aws.amazon.com/gamelift/latest/developerguide/unity-plug-in-scenario.html#unity-plug-in-scenario-boot";
         public const string AwsS3Console = "https://s3.console.aws.amazon.com/s3/home";
         public const string AwsIAMConsole = "https://us-east-1.console.aws.amazon.com/iamv2/home";
-        public const string AwsGameLiftLogs = "https://{0}.console.aws.amazon.com/cloudwatch/home?region={0}#logsV2:logs-insights"; 
+        public const string AwsGameLiftLogs = "https://{0}.console.aws.amazon.com/cloudtrail/home?region={0}#/dashboard"; 
```

Comment:
* Does Cloudtrail contain a better message? 
* If we're already displaying the error message returned is there value to displaying this external link in this case?

Reply:
Might be more of a UX question, but want to understand the use case before talking to UX

Reply:
The link I was provided (the cloudwatch link here) was not showing proper logs, where as cloudtrail shows a log for each of our SDK calls to aws services. 
I like having the external link mainly due to 2 reasons, 
To give users the option to quickly dive deeper into their problems that they come across but not be required. 
Users might not know that the sdk calls are being held in logs (or where they are) so this provides users a gateway to at least see these things.

Reply:
Resolving in favor of a follow up task on our end to sync with UX and confirm this approach is acceptable.